developer_manual/core/theming.rst

```diff
@@ -1,166 +1,183 @@
-
-* Create a version of your logo in .ico format
-* Store your custom favicon as .../owncloud/themes/MyTheme/core/img/favicon.ico
-* Include .../owncloud/themes/MyTheme/core/img/favicon.svg and favicon.png to cover any future updates to favicon handling.
-
-Changing the default colours
-----------------------------
-With a web-developer tool like Mozilla-Inspector, you also get easily displayed the color of the background you clicked on.
-On the top of the login page you can see a case- distinguished setting for different browsers:
+Themes can be used to customize the look and feel of any aspect of an ownCloud installation.
+They can override the default *JavaScript*, *CSS*, *image*, and *template* files, as well as the *user interface translations* with custom versions.
+They can also affect both the web front-end and the ownCloud Desktop client. 
+However, this documentation only covers customizing the web front-end, *so far*.
+
+.. note::
+   Throughout this section of the documentation, for sakes of simplicity, it
+   will be assumed that your owncloud installation directory is ``/owncloud``.
+   If you’re following this guide to create or customise a theme, please make
+   sure you change any references to match the location of your owncloud
+   installation.
+
+How to Create a New Theme
+-------------------------
+
+At its most basic, to create a theme requires two steps:
+
+1. Copy and extend an existing theme or create one from scratch.
+2. Enable the theme in the ownCloud Admin dashboard.
+
+All themes, whether copied or new, must meet two key criteria, these are:
+
+1. They must be located in a directory under the ``apps`` directory of your ownCloud installation.
+2. They require a configuration file called ``appinfo/info.xml`` to be present.
+
+appinfo/info.xml
+~~~~~~~~~~~~~~~~
+
+Here’s an example of the bare minimum which the file needs to contain: 
+
+::
+
+  <?xml version="1.0"?>
+  <info>
+      <id>theme-example</id>
+      <name>Enterprise ownCloud Theme</name>
+      <types>
+          <theme/>
+      </types>
+      <dependencies>
+        <owncloud min-version="9.2" max-version="10" />
+      </dependencies>
+  </info>
+
+And here’s a longer, more complete example:
+
+::
+
+  <?xml version="1.0"?>
+  <info>
+      <id>theme-example</id>
+      <name>Enterprise ownCloud Theme</name>
+      <description>This App provides the enterprise ownCloud theme.</description>
+      <licence>AGPL</licence>
+      <author>Philipp Schaffrath</author>
+      <version>0.0.1</version>
+      <types>
+          <theme/>
+      </types>
+      <dependencies>
+          <owncloud min-version="9.2" max-version="10" />
+      </dependencies>
+  </info>
+
+The value of the ``id`` element needs to be the name of your theme’s folder. We recommend that it always be prefixed with ``theme-``. The main reason for doing so, is that when being viewed in the ownCloud Admin dashboard, it's clear that it’s a theme, not an application. 
```

Comment:
The reason for that, was that it would be alphabetically sorted in a terminal when handling app folders. The view on the admin page sorts by `<name>`.